packages/python/src/components/Class.tsx

```diff
@@ -0,0 +1,30 @@
+import { Children, Indent, List, Scope, Show } from "@alloy-js/core";
+import { usePythonNamePolicy } from "../name-policy.js";
+import { Declaration, DeclarationProps } from "./Declaration.js";
+
+export interface ClassProps extends DeclarationProps {
+  name: string;
+  bases?: Children[];
+}
+
+export function Class(props: ClassProps) {
+  const name = usePythonNamePolicy().getName(props.name, "class");
+  // Determine if children are present
+  const hasChildren =
```

Comment:
use childrenArray to get the children as an array.

Reply:
Fixed.